src/main/java/de/bonndan/nivio/output/LocalServer.java

```diff
@@ -11,11 +13,34 @@
 
     private static Environment env;
 
-    public static String host() {
+    /**
+     * without slash
+     */
+    private final String baseUrl;
+
```

Comment:
Im Wesentlichen wurden jetzt nur die Methoden auf private gesetzt, richtig?

Reply:
Genau, und nicht mehr statisch. Die Methoden geben nur das zurück, was man in de JVM sieht, und das ist meist unbrauchbar für Urls, die im Browser gelinkt werden sollen.